schemas/expconf/v0/log-action.json

```diff
@@ -2,32 +2,25 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "$id": "http://determined.ai/schemas/expconf/v0/log-action.json",
     "title": "LogAction",
-    "$comment": "this is a union of all possible properties, with validation for the common properties",
-    "if": {
-        "required": [
-            "type"
+    "union": {
+        "defaultMessage": "expect one of the followings: cancel_retries, exclude_node, or an legacy object where object[\"type\"] is one of 'cancel_retries' or 'exclude_node'",
```

Comment:
in the error message, don't ever tell users to do the legacy thing.

Shorten it to say `must be one of \"cancel_retries\" or \"exclude_node\"`, and do include the quotes because they clearly indicate the type to the reader.

Reply:
This remind me one thing. I forgot to add a deprecation notice in the trial log if a user submit legacy log_policies.